flycheck.el

```diff
@@ -4608,6 +4627,7 @@ symbols in the command."
           ;; example for such a conflict.
           (setq process (apply 'start-process (format "flycheck-%s" checker)
                                nil command))
+          ;; why is this using setf and not set-process-sentinal ??
```

Comment:
`setf` sets the value of a “generalized variable”.  Essentially it's just a consistent syntax to set many different types of variables, include process sentinels.  If you expand the macro you'll find that it actually expands to `(set-process-sentinel process …)`.

Whether you use generalized variables or not is ultimately a matter of taste; personally I prefer the consistency it gives.

If this answers your question, please remove this comment again :blush: